server/src/test/java/io/unitycatalog/server/base/catalog/BaseCatalogCRUDTest.java

```diff
@@ -55,7 +61,8 @@ public void testCatalogCRUD() throws ApiException {
 
         // Update catalog
         System.out.println("Testing update catalog..");
-        CatalogInfo updatedCatalogInfo = catalogOperations.updateCatalog(CATALOG_NAME, CATALOG_NEW_NAME, CATALOG_NEW_COMMENT);
+        UpdateCatalog updateCatalog = new UpdateCatalog().newName(CATALOG_NEW_NAME).comment(CATALOG_NEW_COMMENT);
+        CatalogInfo updatedCatalogInfo = catalogOperations.updateCatalog(CATALOG_NAME, updateCatalog);
```

Comment:
what about updating properties?

Reply:
Plan on implementing the update part in a follow up PR as this PR was already relatively big